core/spec/models/spree/order_shipping_spec.rb

```diff
@@ -138,7 +138,7 @@ def emails
       before { Spree::OrderCancellations.new(order).short_ship([order.inventory_units.first]) }
 
       it "updates the order shipment state" do
-        expect { subject }.to change { order.reload.shipment_state }.from('ready').to('shipped')
+        expect { subject }.to change { order.reload.shipment_state }.from('pending').to('shipped')
```

Comment:
If this is the case then it was previously broken. Previously the shipment went to the `ready